src/tests/ftest/soak/soak_test_base.py

```diff
@@ -155,6 +161,90 @@ def pre_tear_down(self):
             self.hostlist_clients = [socket.gethostname().split('.', 1)[0]]
         return errors
 
+    def launch_harasser(self, harasser, pool):
+        """Launch any harasser tests if defined in yaml.
+
+        Args:
+            harasser (str): harasser to launch
+            pool (list): list of TestPool obj
+
+        Returns:
+            status_msg(str): pass/fail status message
+
+        """
+        # Init the status message
+        status_msg = None
+        job = None
+        results = multiprocessing.Queue()
+        args = multiprocessing.Queue()
+        # Launch harasser
+        self.log.info("\n<<<Launch harasser %s>>>\n", harasser)
+        if harasser == "snapshot":
+            method = launch_snapshot
+            name = "SNAPSHOT"
+            params = (self, self.pool[0], name)
+            job = threading.Thread(target=method, args=params, name=name)
+        elif harasser == "exclude":
+            method = launch_exclude_reintegrate
+            name = "EXCLUDE"
+            params = (self, pool[1], name, results, args)
+            job = multiprocessing.Process(target=method, args=params, name=name)
```

Comment:
(pylint-redefined-variable-type) Redefinition of job type from threading.Thread to multiprocessing.context.Process